app/controllers/building_map_controller.rb

```diff
@@ -1,5 +1,6 @@
 class BuildingMapController < ApplicationController
-  YOUR_LOCATION_MAGIC_STRING = "Your location".freeze # TODO: This is currenty hard coded in the building_map.js file
+  # TODO: This is currenty hard coded in the building_map.js.erb file
+  YOUR_LOCATION_MAGIC_STRING = "Your location".freeze
```

Comment:
Actually, it is hardcoded in `constants.js`, right?

Anyway, what is this even? I suppose it's related to internationalization? So not part of this issue, but still...?

Reply:
We just moved the comment to the other line. The line became too long for the linter because of the added ending .erb

But I think you are right, this is now in `constants.js`